### PR TITLE
Remove Symfony <4.4 support

### LINKED ISSUE #1629: Drop EOL Symfony support
Symfony ^4.3.0, ^5.0.0 are EOL July 2020.

### PR DESCRIPTION
**Pull request by fbourigault**
Replace #1816 (I accidentally removed the branch...)

Fix #1629.

### CODE REVIEW

File: src/Propel/Generator/Command/InitCommand.php [hunk 1]
```diff
@@ -411,12 +410,8 @@ private function reverseEngineerSchema(OutputInterface $output, array $options)
      */
     protected function createConsoleHelper(InputInterface $input, OutputInterface $output)
     {
-        /* Check if it runs in Symfony3 env — than use QuestionHelper, because DialogHelper is absent */
-        if (class_exists('\Symfony\Component\Console\Helper\QuestionHelper')) {
-            $helper = new ConsoleHelper3($input, $output);
-        } else {
-            $helper = new ConsoleHelper($input, $output);
-        }
+        $helper = new ConsoleHelper3($input, $output);
```
Comment (dereuromark) [inline src/Propel/Generator/Command/InitCommand.php:413]: shouldn't we have the ConsoleHelper used, why "3"?

Comment (fbourigault) [reply src/Propel/Generator/Command/InitCommand.php:413]: The ConsoleHelper was here to support Symfony 2.x (it uses DialogHelper which was removed in 3.0).

IMHO we should remove all the ConsoleHelper stuff and use Symfony helpers.

**Review by dereuromark — APPROVED** [review]

**Comment by fbourigault** [comment]
I kept the `ConsoleHelper` as it contains some useful logic. However the `ConsoleHelperInterface` is gone!

**Comment by dereuromark** [comment]
Any 2nd reviewer for this maybe?

File: src/Propel/Generator/Command/Helper/ConsoleHelper.php [hunk 2]
```diff
@@ -158,4 +160,19 @@ public function writeln($messages, $options = 0)
...
+     * @param string $question
+     * @param string|null $default
+     *
+     * @return string
+     */
+    private function formatQuestion($question, $default = null)
+    {
+        if ($default) {
+            return sprintf('<info>%s</info> [<comment>%s</comment>]: ', $question, $default);
+        } else {
```
Comment (gechetspr) [inline src/Propel/Generator/Command/Helper/ConsoleHelper.php:174]: Not a blocker, but we don't need this else statement. We can just return a string without default after or even we can prepare it beforehand and add default part if default is not null.

**Review by gechetspr — APPROVED** [review]
All good except one minor thing that is just a style issue

File: src/Propel/Generator/Command/Helper/ConsoleHelper.php (hunk 2 above)
Comment (dereuromark) [reply src/Propel/Generator/Command/Helper/ConsoleHelper.php:174]: The else can indeed be removed, just return afterwards without else.

Comment (fbourigault) [reply src/Propel/Generator/Command/Helper/ConsoleHelper.php:174]: TBH, I didn't do anything to this class but renaming it from `ConsoleHelper3` to `ConsoleHelper`.

Do you expect I improve it?

Comment (dereuromark) [reply src/Propel/Generator/Command/Helper/ConsoleHelper.php:174]: Improving is always nice, but no requirement here.

**Pull request merged**